src/api/BdmsContextExtensions.cs

```diff
@@ -85,9 +86,9 @@ public static void SeedData(this BdmsContext context)
 
         // Seed Boreholes
         var borehole_ids = 1000;
```

Comment:
Wenn wir das Konzept beibehalten wollen,  dass  die ID der Entities in jeder Tabelle mit einer neuen Zahl beginnt müssten wir an die Startwerte ein paar Nullen anhängen.

Reply:
Stimmt, das sollten wir entsprechend anpassen. Das mache ich noch in diesem PR.